tests/containers/container_diff.pm

```diff
@@ -21,16 +21,15 @@ sub run {
 
     zypper_call("install container-diff") if (script_run("which container-diff") != 0);
 
-    my ($testing_images, $released_images) = get_suse_container_urls();
+    my $unreleased_image = get_image_url(released => 0);
+    my $released_image = get_image_url();
     # container-diff
-    for my $i (@{$testing_images}) {
-        my $image_file = $testing_images->[$i] =~ s/\/|:/-/gr;
-        my $container_diff_results = "/tmp/container-diff-$image_file.txt";
-        assert_script_run("docker pull $testing_images->[$i]", 360);
-        assert_script_run("docker pull $released_images->[$i]", 360);
-        assert_script_run("container-diff diff daemon://$testing_images->[$i] daemon://$released_images->[$i] --type=rpm --type=file --type=size > $container_diff_results", 300);
-        upload_logs("$container_diff_results");
-    }
+    my $image_file = $unreleased_image =~ s/\/|:/-/gr;
+    my $container_diff_results = "/tmp/container-diff-$image_file.txt";
+    assert_script_run("docker pull $unreleased_image", 360);
```

Comment:
`docker pull` smells like a good candidate for `script_retry` ....

```suggestion
    script_retry("docker pull $unreleased_image", timeout => 360, retries => 3, delay => 120);
```

Same for the next line

Reply:
Yes, let's leave this for separate pull request please.